bin/index.js

```diff
@@ -40,6 +40,28 @@ program
       });
   });
 
+program
+  .command('from-build-config [siteDir]')
```

Comment:
This look different than at `from-build`:
https://github.com/KohheePeace/docusaurus-pdf/blob/d90109638a5c98e6a956af3cf550c01b176853cf/bin/index.js#L26

It seems you know how to deal with commander.js. Please feel free to fix the command of `from-build` to have an uniform API across the different commands.

Reply:
Ah, yes, that is my fault :smile: I was trying to copy the `docusaurus build` API, but I think it makes more sense to copy the `from-build` API.

I have changed `from-build` slightly to use commander.js's default option value, and specified that `--output-file` must have an argument. I'm not sure if I should change the API anymore, since there are a few people using this package according to npm, and changing the API might break their scripts.

I've also changed `from-build-config` to match `from-build` more. Now `dirPath` is an argument, and `siteDir` is an option.

Reply:
You're right! I totally forgot that there are already some users using this project! We should avoid changing the API unless it is absolutely necessary. 😊

Thank's for the fix with the `--output-file` argument 👌